recipe/meta.yaml

```diff
@@ -126,21 +126,28 @@ outputs:
         - echo {{ rust_arch }}
 
   - name: rust-src
-    script: install-rust-src.sh
+    script: install-rust-src.sh  # [unix]
+    script: install-rust-src.bat  # [win]
     build:
-      noarch: generic
-      skip: {{ rust_arch != "x86_64-unknown-linux-gnu" }}
+      skip: false
```

Comment:
Ah, it is too bad that we are losing the `noarch`-ness of the `rust-src` package.

Reply:
I’m sorry too. This is the simplest solution I can think of right now. If you have a better idea, please feel free to share it.